src/Client.js

```diff
@@ -170,7 +170,12 @@ class Client extends EventEmitter {
         }
 
         await page.evaluate(ExposeStore, moduleRaid.toString());
-
+        
+        // remember me
+        await page.evaluateOnNewDocument(() => {
```

Comment:
I'm not sure this would work here- `evaluateOnNewDocument` will run when there's some kind of navigation, which isn't the case. Should this be set before navigating to `WhatsWebURL`?

Reply:
> I'm not sure this would work here- `evaluateOnNewDocument` will run when there's some kind of navigation, which isn't the case. Should this be set before navigating to `WhatsWebURL`?

we do it the same way we do the legacy session

Reply:
You are right

Reply:
fixed